core/renderers/block_rendering_rewrite/block_render_draw_highlight.js

```diff
@@ -50,176 +50,178 @@ goog.require('Blockly.blockRendering.Measurable');
 Blockly.blockRendering.Highlighter = function(info, pathObject) {
   this.info_ = info;
   this.pathObject_ = pathObject;
-  this.highlightSteps_ = this.pathObject_.highlightSteps;
-  this.highlightInlineSteps_ = this.pathObject_.highlightInlineSteps;
+  this.steps_ = this.pathObject_.highlightSteps;
+  this.inlineSteps_ = this.pathObject_.highlightInlineSteps;
+
+  this.RTL_ = this.info_.RTL;
+
+  /**
+   * The offset between the block's main path and highlight path.
+   * @type {number}
+   * @private
+   */
+  this.highlightOffset_ = Blockly.blockRendering.highlightConstants.OFFSET;
+
+  this.outsideCornerPaths_ = Blockly.blockRendering.highlightConstants.OUTSIDE_CORNER;
+  this.insideCornerPaths_ = Blockly.blockRendering.highlightConstants.INSIDE_CORNER;
+  this.puzzleTabPaths_ = Blockly.blockRendering.highlightConstants.PUZZLE_TAB;
+  this.notchPaths_ = Blockly.blockRendering.highlightConstants.NOTCH;
+  this.startPaths_ = Blockly.blockRendering.highlightConstants.START_HAT;
 };
 
 Blockly.blockRendering.Highlighter.prototype.drawTopCorner = function(row) {
   for (var i = 0, elem; elem = row.elements[i]; i++) {
     if (elem.type === 'square corner') {
-      this.highlightSteps_.push(Blockly.blockRendering.constants.START_POINT_HIGHLIGHT);
+      this.steps_.push(Blockly.blockRendering.highlightConstants.START_POINT);
```

Comment:
What's the rule for when you create a property on this vs referring to something in blockRendering directly?